applications/osb-portal/src/pages/RepositoryPage.tsx

```diff
@@ -45,6 +47,11 @@ import { CodeBranchIcon } from "../components/icons";
 
 
 const useStyles = makeStyles((theme) => ({
+  infoIcon: {
+    fontSize: "0.9rem",
+    verticalAlign: "middle",
```

Comment:
This doesn't work

Reply:
![osbv2-vertical-align](https://user-images.githubusercontent.com/102575/152810675-97edcea4-1ef5-4d35-bfdd-05bfc31ce68d.png)


It does seem to do something, but maybe not what is needed. If I deselect it, the icon moves up vertically ^.

Let me add the Box etc. and see if that makes a difference here.